temperature/crud.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+import httpx
+from fastapi import HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from . import models
+
+
+async def fetch_current_temperature(city_name: str) -> float:
+    """
+    Fetches current temperature for a given city from MetaWeather API.
+    """
+    # Запрос для получения WOEID города
+    location_url = f"https://www.metaweather.com/api/location/search/?query={city_name}"
+    async with httpx.AsyncClient() as client:
+        location_response = await client.get(location_url)
+        if location_response.status_code != 200 or not location_response.json():
+            raise HTTPException(status_code=404, detail=f"City '{city_name}' not found.")
```

Comment:
Consider handling potential exceptions from the `httpx` requests more gracefully, such as network errors or unexpected response formats, to improve robustness.